src/modules/auth/views/Register.vue

```diff
@@ -11,26 +11,64 @@ const router = useRouter();
 const email = ref("");
 const password = ref("");
 
+const passwordtest = ref("");
+const passwordType = ref ("password")
+
 async function handleSignUp() {
-  const result = await auth.signIn(email.value, password.value);
+  const result = await auth.signUp(email.value, password.value);
   if (!result) {
-    alert("email ou senha incorretos");
+    alert("email ou senha invalidos");
     return;
   }
   router.push("/login");
 }
 
+const confirmPassword = computed(() => {
+  return password.value === passwordtest.value;
+});
```

Comment:
a ideia dessa função é excelente! só dando uma limpada, mas nada de errado
```suggestion
const confirmPassword = computed(() =>
    password.value === passwordtest.value
);
```